### ISSUE TITLE
Accurate current Location is invalid 

### ISSUE THREAD

**Issue by khaboshama91** [BirjuVachhani/locus-android#58]
**Describe the bug**
The library doesn't support to get user current location and it's depend on google map location to retrieve the last known location, please can you solve it
@BirjuVachhani

**To Reproduce**
1- open app
2- request to get current location (Locus.getCurrentLocation())

**Expected behavior**
I expected to retrieve the user current location instead of the last known location

**Smartphone (please complete the following information):**
- Any version

**Comment by rodneyrd** [comment]
up

**Comment by tkasymuulu** [comment]
up

**Comment by amensia** [comment]
up ;)

**Comment by rahulas** [comment]
Any update on this?

**Comment by rahulas** [comment]
How does this work? Any explanation pls.

**Comment by rahulas** [comment]
Tried it but, it doesn't work. Still, showing wrong location.

**Comment by ichsanudin98** [comment]
up

**Comment by amensia** [comment]
My solution is to trigger 2 update instead 
`        Locus.configure {
            request {
                this.numUpdates = 2
                this.priority = LocationRequest.PRIORITY_HIGH_ACCURACY
                this.interval = 100
            }
        }
`

**Comment by BirjuVachhani** [comment]
This should be fixed in `v4.1.0`. Feel free to reopen if it is not.

```groovy
implementation 'com.github.BirjuVachhani:locus-android:4.1.0'
```
